exosphere-shared/templates/add-service/exoservice-ls-mongodb/package.json

```diff
@@ -4,7 +4,8 @@
   "dependencies": {
     "exoservice": "0.21.8",
     "get-env": "0.5.10",
-    "mongodb": "2.2.16"
+    "mongodb": "2.2.16",
+    "nitroglycerin": "1.1.2"
```

Comment:
This is being picky, but in the future if this wasn't failing tests then it can be extracted into a separate PR